storagenode/console/service.go

```diff
@@ -97,117 +102,139 @@ func NewService(log *zap.Logger, consoleDB DB, bandwidth bandwidth.DB, pieceStor
+	DiskSpace DiskSpaceInfo `json:"diskSpace"`
+	Bandwidth BandwidthInfo `json:"bandwidth"`
 
-	return s.consoleDB.Bandwidth().GetDailyTotal(ctx, from, to)
+	Version  version.SemVer `json:"version"`
+	UpToDate bool           `json:"upToDate"`
 }
 
-// GetDailyBandwidthUsed returns slice of daily bandwidth usage for provided time range,
-// sorted in ascending order for particular satellite
-func (s *Service) GetDailyBandwidthUsed(ctx context.Context, satelliteID storj.NodeID, from, to time.Time) (_ []BandwidthUsed, err error) {
+// GetDashboardData returns stale dashboard data
+func (s *Service) GetDashboardData(ctx context.Context) (_ *Dashboard, err error) {
 	defer mon.Task()(&ctx)(&err)
+	data := new(Dashboard)
 
-	return s.consoleDB.Bandwidth().GetDaily(ctx, satelliteID, from, to)
-}
+	data.NodeID = s.kademlia.Local().Id
+	data.Wallet = s.walletAddress
+	data.Version = s.versionInfo.Version
+	data.UpToDate = s.version.IsAllowed()
+	data.Satellites = s.trust.GetSatellites(ctx)
 
-// GetBandwidthBySatellite returns all info about storage node bandwidth usage by satellite
-func (s *Service) GetBandwidthBySatellite(ctx context.Context, satelliteID storj.NodeID) (_ *BandwidthInfo, err error) {
-	defer mon.Task()(&ctx)(&err)
+	spaceUsege, err := s.pieceStore.SpaceUsedForPieces(ctx)
+	if err != nil {
+		return nil, SNOServiceErr.Wrap(err)
+	}
 
-	summaries, err := s.bandwidthDB.SummaryBySatellite(ctx, time.Time{}, time.Now())
+	bandwidthUsage, err := bandwidth.TotalMonthlySummary(ctx, s.bandwidthDB)
 	if err != nil {
-		return nil, err
+		return nil, SNOServiceErr.Wrap(err)
 	}
 
-	// TODO: update bandwidth.DB with GetBySatellite
-	return FromUsage(summaries[satelliteID], s.allocatedBandwidth.Int64())
+	data.DiskSpace = DiskSpaceInfo{
+		Used:      memory.Size(spaceUsege).GB(),
+		Available: s.allocatedDiskSpace.GB(),
+	}
+
+	data.Bandwidth = BandwidthInfo{
+		Egress: Egress{
+			Repair: bandwidthUsage.GetRepair,
+			Audit:  bandwidthUsage.GetAudit,
+			Usage:  bandwidthUsage.Get,
+		},
+		Ingress: Ingress{
+			Repair: bandwidthUsage.PutRepair,
+			Usage:  bandwidthUsage.Put,
+		},
+		Used:      memory.Size(bandwidthUsage.Total()).GB(),
+		Available: s.allocatedBandwidth.GB(),
+	}
+
+	return data, nil
+}
+
+// Satellite encapsulates satellite related data
+type Satellite struct {
+	ID             storj.NodeID         `json:"satelliteID"`
+	StorageDaily   []storageusage.Stamp `json:"storageDaily"`
+	BandwidthDaily []BandwidthUsed      `json:"bandwidthDaily"`
+	Audit          reputation.Metric    `json:"audit"`
+	Uptime         reputation.Metric    `json:"uptime"`
 }
 
-// GetUsedStorageTotal returns all info about storagenode disk space usage
-func (s *Service) GetUsedStorageTotal(ctx context.Context) (_ *DiskSpaceInfo, err error) {
+// GetSatelliteData returns satellite related data
+func (s *Service) GetSatelliteData(ctx context.Context, satelliteID storj.NodeID) (_ *Satellite, err error) {
 	defer mon.Task()(&ctx)(&err)
+	// get current month edges
```

Comment:
I don't think that this comment is needed as the `date.MonthBoundary` function documentation already mentions what does this.